### DISCUSSION TITLE
Esse app é "seguro" para usar em um site com checkout em produção?

### DISCUSSION THREAD

**Discussion by thiagochirana** [andreroggeri/pynubank#434] [Q&A] [answered]
Gostaria de saber essa informação, se tem casos aqui que usa isso em produção, pois até onde sei o Nubank não tem uma api de desenvolvedores.

**Comment by andreroggeri** [comment, marked as answer]
Não recomendo.
Não por não ser seguro, mas isso é uma engenharia reversa que amanhã pode parar de funcionar (Vide o acesso a conta corrente que atualmente está complicado).
